Client/Application/LeanplumIntegration.swift

```diff
@@ -135,6 +135,12 @@ class LeanplumIntegration {
         }
     }
 
+    func track(eventName: LeanplumEventName, withInfo: String) {
+        if shouldSendToLP() {
+            Leanplum.track(eventName.rawValue, withInfo: withInfo)
```

Comment:
Yes, this was my oversight - it's why I had @bkmunar fake a map with a string 'Source: Something'. @bkmunar please go back and change this to a map instead of a info string.